### PR TITLE
Fix #2118: Feature Request > Indicate font size next to font size UI

### LINKED ISSUE #2118: Feature Request > Indicate font size next to font size UI
For this option...

image

Would be nice to indicate the actual font size you're setting (or percentage), so that you can set it to something you like (and remember) more easily.

### PR DESCRIPTION
**Pull request by azusaaz**
Display font size
textsize

(On the clip, the letter "あ" shows up, I could not remove it, but it's my windows' bug)

### CODE REVIEW

File: public/editor/scripts/ui/menus.js [hunk 1]
```diff
@@ -126,10 +126,17 @@ function setupOptionsMenu(bramble) {
     "#editor-pane-nav-options-menu"
   );
 
-  // Font size
+  // Display font size
+  $(document).ready(function(){
```
Comment (humphd) [inline public/editor/scripts/ui/menus.js]: The Travis build is complaining about this line:

```
/home/travis/build/mozilla/thimble.mozilla.org/public/editor/scripts/ui/menus.js
  130:31  error  Insert `·`  prettier/prettier
```

@gideonthomas what is the issue here?

Comment (humphd) [reply public/editor/scripts/ui/menus.js]: @azusaaz for future, when you have an automated build fail like this, you can click the `Details` link in your PR to have it show you what's wrong:

https://travis-ci.org/mozilla/thimble.mozilla.org/builds/289327030#L483

Comment (humphd) [reply public/editor/scripts/ui/menus.js]: Also, the AppVeyor build is failing with the same problem, so fixing it once will fix both builds:

https://ci.appveyor.com/project/mozillafoundation/thimble-mozilla-org/build/77#L1113

Comment (azusaaz) [reply public/editor/scripts/ui/menus.js]: @humphd  Thank you! I'll check the line.

**Comment by humphd** [comment]
Great job @azusaaz, that fixed it.

**Comment by azusaaz** [comment]
@humphd  Thank you !!!

**Comment by gideonthomas** [comment]
who would this require a re-review from? @flukeout?

**Comment by flukeout** [comment]
I'll test it out now.

**Comment by flukeout** [comment]
Great, this works as expected. If the code checks out, please go ahead and merge @gideonthomas or @humphd - thanks a lot @azusaaz

**Comment by azusaaz** [comment]
@flukeout Thank you for the check!
@humphd Oh, ok. I've thought other scenarios will accompany with the refresh and ".ready" can cover the change. but seems not enough. I'll fix that. Thank you!

**Comment by azusaaz** [comment]
The trigger is fontchange itself this time.

**Comment by azusaaz** [comment]
The trigger is font change itself this time.

File: public/editor/scripts/ui/menus.js [hunk 2]
```diff
@@ -126,7 +126,19 @@ function setupOptionsMenu(bramble) {
     "#editor-pane-nav-options-menu"
   );
 
-  // Font size
+  // Display Font size
+  $("#editor-pane-nav-font-size").ready(function() {
```
Comment (humphd) [inline public/editor/scripts/ui/menus.js]: I don't think this is needed (i.e., the `ready` handler).  Instead, let's just extract the function you have on 136-139 (i.e., the callback) and make it a function we can also call on startup here:

```js
// Whenever the font size changes in Brackets, update what we display in Thimble.
function updateFontSize() {
  var fontSize = bramble.getFontSize();
  $("#editor-pane-nav-font-size").text(fontSize);
}

bramble.on("fontSizeChange", updateFontSize);

// Set the initial font size on startup
updateFontSize();
```

Comment (azusaaz) [reply public/editor/scripts/ui/menus.js]: ok. Thank you!

**Comment by humphd** [comment]
@azusaaz merged!  Thanks for working on this, it's a nice improvement.  Welcome to Thimble.

**Comment by azusaaz** [comment]
@humphd  Thank you for your help!

**Pull request merged**